src/purging_rmw.c

```diff
@@ -245,18 +245,23 @@ get_then_time(struct dirent *entry, const char *entry_path)
  * @see get_then_time
  */
 int
-purge (const st_waste * waste_curr)
+purge (const st_waste * waste_curr, const rmw_options * cli_user_options)
 {
   short status = 0;
 
-  bool cmd_empty = 0;
   bool cmd_dry_run = 0;
   char *rmwtrash_env = getenv("RMWTRASH");
 
   if (rmwtrash_env != NULL)
-  {
-    cmd_empty = strcmp (rmwtrash_env, "empty") ? 0 : 1;
     cmd_dry_run = strcmp (rmwtrash_env, "dry-run") ? 0 : 1;
+
+  if (cli_user_options->want_empty_trash) {
+    puts(_("All the files in trash will be unrecoverable after empty."));
```

Comment:
Technically they could be recovered by some utilities such as https://linuxgain.com/recover-deleted-files-linux/
```suggestion
    puts(_("All waste folders and their contents will be erased -"));
```

Reply:
Applied this change.